blog/_posts/2022-09-29-scala-survey-2022-announcement.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: blog-detail
+post-type: blog
+by: Kacper Korban, VirtusLab
+title: Participate in the Scala Survey 2022!
+---
+
+![Scala Survey 2022 Announcement]({{ site.baseurl }}/resources/img/scala-survey-2022-announcement.png)
+
+## The Scala Team wants to hear from you!
+
+We want to make the language we all love as amazing as possible. We want to know which way the Scala community is moving and which parts of the Scala language you utilize the most. That’s why we invite you to help by participating in the Scala Survey 2022.
+
+[Scala Survey 2022](https://scala-survey-2022.virtuslab.com/)
+
+The Survey will close on **21.10.2022**.
+
+## About the survey
+
+Completing the survey should take 4-12 minutes. The survey is anonymous, and most of the questions are optional, so if you don’t have that much time, you don’t have to answer all of them.
+
+**Who is the survey intended for?** Everyone. Whether you’re an experienced Scala Developer or just starting your Scala journey, we count on your participation.
+
+After the survey closes, we will publish the aggregated results on the [Scala blog](https://www.scala-lang.org/blog/).
```

Comment:
Could we have a line here about the content of the survey's questions, i.e overview of the topics covered

Reply:
e.g. We are interested in learning what libraries and tools people are using, what platforms they support, the future direction of tooling focus, where you are learning to use scala/training engineers, etc.